### DISCUSSION TITLE
How to mass update current aliases with an additional or different current mailbox?

### DISCUSSION THREAD

**Discussion by zahroc** [simple-login/app#459] [Q&A]
I wanted to start sending emails to two different mailboxes. I made the change in the mailbox and domain area but it seems current aliases do not update with the change. Am I missing something? Is there a way to mass update the current aliases?

**Comment by nguyenkims** [comment]
Hey updating the mailboxes for a custom domain only affect aliases created with the catch-all. The bulk editing aliases feature is currently tracked in this thread https://github.com/simple-login/app/discussions/407 :)
